CaptureSDK/GiniCaptureSDK/Sources/GiniCaptureSDK/Resources/de.lproj/Localizable.strings

```diff
@@ -19,6 +19,7 @@
 "ginicapture.navigationbar.analysis.back"       = "Abbrechen";
 "ginicapture.navigationbar.help.backToCamera"   = "";
 "ginicapture.navigationbar.help.backToMenu"     = "Hilfe";
+"ginicapture.navigationbar.analysis.backToReview" = "TODO Review";
```

Comment:
[Suggestion] If it is the same wording as the Review screen title, you can find that here in the list.

Reply:
@vizaknaidavid I don't think we need to reuse it. We allow overriding all labels, it will be safer to keep separate localized strings.
Which strings do you suggest reusing? 🧐

Reply:
I was referring just for the translation. Because the word "Review" is already translated in the list

Reply:
@zladzeyka @vizaknaidavid   I have update the "ginicapture.navigationbar.analysis.backToReview" to "Übersicht"